plonetheme/onegov/tests/test_navigation_portlet.py

```diff
@@ -122,6 +124,7 @@ def test_lists_siblings_when_configured(self, browser):
         self.assertEquals(['Plone site', 'Foo', 'Bar', 'Baz'],
                           portlet().css('li').text)
 
+    @skipIf(IS_PLONE_5, 'Plone 5 renders the navigation portlet twice.')
```

Comment:
I take this does not happen in vanilla Plone 5.1. Where does this second render happen? Only in tests?

Reply:
This happened only in tests and was caused by `plone.app.contenttypes` which adds the navigation portlet.